<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaestralSettings</class>
 <widget class="QWidget" name="MaestralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>457</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Maestral Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayoutMain" columnstretch="4,4,3">
     <item row="7" column="1">
      <widget class="CustomCombobox" name="comboBoxUpdateInterval">
       <item>
        <property name="text">
         <string>Daily</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weekly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monthly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Never</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelExcludedFoldersTitle">
       <property name="text">
        <string>Selective sync:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QWidget" name="widgetCLI" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="Line" name="line0">
       <property name="styleSheet">
        <string notr="true">color: rgb(213, 213, 213)</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonUnlink">
       <property name="text">
        <string>Unlink this Dropbox...</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="checkBoxNotifications">
       <property name="text">
        <string>Enable notifications</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelDropboxPathTitle">
       <property name="text">
        <string>Local Dropbox folder:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="labelAboutTitle">
       <property name="text">
        <string>About Maestral:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelSystemSettings">
       <property name="text">
        <string>System settings:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="Line" name="line1">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(213, 213, 213)</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayoutAccountInfo">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="labelAccountInfo">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mail@outlook.com, Dropbox Business</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSpaceUsage">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>31.8% of 1,324,980GB used</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" rowspan="3" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelUserProfilePic">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelUpdateInterval">
       <property name="text">
        <string>Check for updates:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButtonExcludedFolders">
       <property name="text">
        <string>Select files and folders...</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="checkBoxStartup">
       <property name="text">
        <string>Start Maestral on login</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="labelAccountName">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>User Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="12" column="1" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="labelUrl">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;{0}&quot;&gt;{0}&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelVersion">
         <property name="text">
          <string>GUI v{0}, daemon v{1}</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelCopyright">
         <property name="styleSheet">
          <string notr="true">color: rgb(127, 127, 127)</string>
         </property>
         <property name="text">
          <string>Â© 2018-{0}, {1}.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="Line" name="line2">
       <property name="styleSheet">
        <string notr="true">color: rgb(213, 213, 213)</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="CustomCombobox" name="comboBoxDropboxPath"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomCombobox</class>
   <extends>QComboBox</extends>
   <header>maestral_qt.widgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonUnlink</tabstop>
  <tabstop>pushButtonExcludedFolders</tabstop>
  <tabstop>comboBoxDropboxPath</tabstop>
  <tabstop>comboBoxUpdateInterval</tabstop>
  <tabstop>checkBoxStartup</tabstop>
  <tabstop>checkBoxNotifications</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
